SKILL.tsv

```diff
@@ -5096,7 +5096,7 @@ SKILL_20150918_005095	{memo X}$Corpe Tower Duration: #{CaptionRatio}# seconds{nl
 SKILL_20150918_005096	$Raise Dead
 SKILL_20150918_005097	${#993399}{ol}[Magic]{/}{/}Summons a skeleton soldier made with a corpse. The skeleton soldier will engage in combat and follow its master.
 SKILL_20150918_005098	$Summons #{CaptionRatio}# Skeleton Soldiers{nl}Consumes #{CaptionRatio2}# Corpse(s){nl}Maximum Level: 5
-SKILL_20150918_005099	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Attack your enemy with a spiral arrow inflicting multiple strikes. 
+SKILL_20150918_005099	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Attack your enemy with a spiraling arrow inflicting multiple strikes. 
```

Comment:
Attack your enemy with a spiraling arrow inflicting multiple strikes. -> Attacks the enemy with a spiraling arrow inflicting multiple strikes.